package.json

```diff
@@ -72,10 +73,10 @@
 		"lodash": "4.17.21",
 		"postcss": "8.4.16",
 		"postcss-cli": "10.0.0",
-		"postcss-loader": "^4.3.0",
+		"postcss-loader": "4.3.0",
 		"prettier": "2.3.2",
 		"react": "16.13.1",
-		"react-bootstrap": "1.6.1",
+		"react-bootstrap": "github:mattermost/react-bootstrap#c6957962364e0818a51bbfd13e89919903b422d6",
```

Comment:
why this change?

Reply:
This change was visible because this branch was not updated. Fixed Now.